Public/Src/FrontEnd/Script/RuntimeModel/AstBridge/AstConverter.cs

```diff
@@ -1364,7 +1364,7 @@ private TypeParameter ConvertTypeParameter(ITypeParameterDeclaration typeParamet
                 var name = SymbolAtom.Create(StringTable, nameStr);
 
                 // It is ok if type is null here
-                var type = ConvertType(declaration.Type, currentQualifierSpaceId);
+                Type type = m_conversionConfiguration.UnsafeOptions.SkipTypeConversion ? null : ConvertType(declaration.Type, currentQualifierSpaceId);

```

Comment:
I'm actually curious why we do the type conversion at all....

Typecheck has passed so why do the runtime one?

Reply:
At that time there's only 1 AST, eval AST. I guess you guys tried to maintain the structure as much as possible.